templates/modules/common/subscription-details.hypr.live

```diff
@@ -0,0 +1,63 @@
+<div class="mz-multiship-email" style="width: 100%;">
+    <div class="mz-table-header" style="padding:15px">
+        <span style="font-size:25px">{{ labels.subscriptionDetails }}</span>
+    </div>
+</div>
+
+<div class="mz-store-details">
+    <div>
+        <div><strong> {{ labels.subscriptionNumber }}</strong> : {{ model.number }}</div>
+        <div><strong> {{ labels.subscriptionStatus }}</strong> : {{ model.status }} </div>
+        <div><strong> {{ labels.frequency }}</strong>: {{model.frequency.value}} {{model.frequency.unit|lower}} </div>
+        <div><strong> {{ labels.nextOrderDate }}</strong>: {{model.nextOrderDate|date("m/d/y")}}</div>
+    </div>
+</div>
+<hr/>
+ <br/>
+    <div>
+        <div>
+            <strong>
+                {{labels.shipTo}} :
+            </strong>
+            <br/><br/>
+            <div class="mz-addresssummary">{% include "modules/common/address-summary" with model=model.payment.billingInfo.billingContact %}</div>
+        </div>
+    </div>
+    <br/></br/>
+    <div>
+        <strong>
+        {{labels.shippingMethod}}:
+        </strong> 
+        {{model.fulfillmentInfo.shippingMethodName}}    
+    </div>
```

Comment:
Display this only if the shipping method is STH.

Reply:
@kibo-himanshivarshney You will need to look into how details are displayed in case of orders for STH and BOPIS and implement things similarly

Reply:
removed the header part.

Reply:
fixed the review comment and got it reviewed by Sagar